core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/contexts/AllContexts.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2021-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.eventgen.protocol.contexts
+
+import com.snowplowanalytics.snowplow.eventgen.protocol.SelfDescribingJsonGen
+
+object AllContexts {
+
+  val derivedContexts: List[SelfDescribingJsonGen] =
+    List(
+      UaParserContext,
+      YauaaContext
+    )
+
+  // "sent" meaning not derived contexts
+  val sentContexts: List[SelfDescribingJsonGen] =
```

Comment:
I think even with `trackerContexts` or `standardContexts` I would still need a code comment to explain to me what it meant.  Do we have a common snowplow word to describe this type of context?  It seems like an important concept, but I don't remember ever hearing it named.

Reply:
Yeah good point. I don't think we do have a term for this.

Best I can think of here is to call them co/cx contexts, because those are the 2 tracker protocol fields that can contain them. But that's even less intuitive a name for this. 

`trackerGeneratedContexts` and `derivedContexts` is the best I can think of in that case. I think maybe that has a better chance of someone getting it straight away, but I'd still leave a comment so it's much of a muchness. I'm fine leaving it as `sent`, us not having a name for that concept isn't really a problem for this PR to get hung up on. :)